addons/scopes/initSettings.inc.sqf

```diff
@@ -91,3 +91,11 @@ private _category = format ["ACE %1", localize LSTRING(DisplayName)];
     false,
     1
 ] call CBA_fnc_addSetting;
+
+[
+    QGVAR(inScopeAdjustment), "CHECKBOX",
+    LSTRING(inScopeAdjustment_displayName),
+    _category,
+    false,
+    0
```

Comment:
Making this "UI preference" global forces the server's setting on everyone, I don't think that's what @PabstMirror intended.

> Maybe it's best to make this a global setting, defaulting off.
That leaves no change to default behavior
**And only people who chose to enable it will see it**

I think state `2` is ideal for this sort of setting, which prevents a server owner/mission creator from overriding a setting that only affects the individual player client who changed it (i.e: not fading out the TFAR voice volume indicator).
State `0` should be fine too, just like `ace_scopes_useLegacyUI`, though IMO that too should be set to `2`.